app/views/providers/start/index.html.erb

```diff
@@ -22,21 +22,4 @@
                 class: 'govuk-button govuk-button--start',
                 id: 'start' %>
   </div>
-
-  <div class="govuk-grid-column-one-third">
```

Comment:
This may be a question for @RoseSAK - if we remove a column one-third, should the other column-two-thirds be updated/renamed?

Reply:
I wasnt sure about that but as removing the 1/3 didnt break the formatting for the rest if the page I left it as is, but I am happy to defer to @RoseSAK

Reply:
Heya, I think since the one-third column is removed, the remaining column should be changed to 'govuk-grid-column-full' rather than left as two-thirds, if that's alright, just to be consistent

Reply:
@willc-work